### PR TITLE
Diff aggregation

### PR DESCRIPTION
**Pull request by mhspelt**
(paraphrased from #600)
Hi, this is my first pull request ever, so I hope I did everything right.
I have a sensor in HA that can go negative during the day (it measures net power usage taking solar into account) and I wanted to know the difference between the first reading of the day and the final reading of the day. So I _added `diff`_ to calculate this (for monotonously increasing data the result is identical)

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
@mhspelt please rebase.

**Comment by mhspelt** [comment]
I thought that was what I did with #601. As a Mercurial user I'm still a
bit confused by git's idea of branches, can you help me out a bit please?

**Comment by kalkih** [comment]
Please add documentation for this: https://github.com/kalkih/mini-graph-card/tree/dev#aggregate-functions

**Comment by jlsjonas** [comment]
@mhspelt Could you please add documentation for this feature as requested?
Thanks for your contribution! (we'll take care of rebasing during merge, as we understand you're unfamiliar with the process)

**Review by jlsjonas — CHANGES_REQUESTED** [review]
Could you please add documentation for this feature?

**Comment by mhspelt** [comment]
Hi @jlsjonas, I've added and pushed to https://github.com/mhspelt/mini-graph-card
I don't really see the commit in this PR yet, does it synchronize automatically?

File: src/graph.js [hunk 1]
```diff
@@ -246,8 +247,12 @@ export default class Graph {
     return this._maximum(items) - this._minimum(items);
   }
 
+  _diff(items) {
+    return this._last(items) - this._first(items);
+  }
+
   _lastValue(items) {
-    if (this.aggregateFuncName === 'delta') {
+    if (this.aggregateFuncName === 'delta' || this.aggregateFuncName === 'diff') {
```
Comment (jlsjonas) [inline src/graph.js:255]:
```suggestion
    if (['delta', 'diff'].includes(this.aggregateFuncName)) {
```
For improved readability I'd suggest this small change, thanks for the contribution!

**Review by jlsjonas — APPROVED** [review]

**Pull request merged**